kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/test/config/ResolvedTestConfig.kt

```diff
@@ -81,6 +83,9 @@ data class ResolvedTestConfig(
       require(threads > 0) { "Number of threads must be greater than 0" }
    }
 
+   // note: the invocation timeout can't be larger than the test case timeout
+   val resolvedInvocationTimeout: Duration = minOf(timeout, invocationTimeout)
```

Comment:
Feel free to propose a better naming for this field. Maybe, it should be internal.

Reply:
Removed this field